index.html

```diff
@@ -2,10 +2,114 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <link rel="stylesheet" href="./styles.css" />
-    <title>Tribute Page Review</title>
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/styles.css" />
   </head>
-  <body></body>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <div class="container">
+        <h1 id="title">Dr. Norman Borlaug</h1>
+        <h2>The man who saved a billion lives</h2>
+      </div>
+
+      <section>
+        <div class="container_a">
+          <img src="./css/image1/tribute-page-main-image.jpg" alt="" />
+          <h3 id="img_description">
+            Dr. Norman Borlaug, third from the left, trains biologists in Mexico
+            on how to increase wheat yields - part of his life-long war on
+            hunger.
+          </h3>
+        </div>
+      </section>
+
+      <article>
+        <div class="container_b">
+          <h5>Here's a time line of Dr. Borlaug's life:</h5>
+          <ul>
+            <li>1914 - Born in Cresco, Iowa</li>
+            <li>
+              1933 - Leaves his family's farm to attend the University of
+              Minnesota, thanks to a Depression era program known as the
+              "National Youth Administration"
+            </li>
+            <li>
+              1935 - Has to stop school and save up more money. Works in the
+              Civilian Conservation Corps, helping starving Americans. "I saw
+              how food changed them", he said. "All of this left scars on me."
+            </li>
+            <li>
```

Comment:
Please use the li tag appropriately by adding a p.tag inside the li.tag as it is best HTML practices.